### DISCUSSION TITLE
Localisation: Swedish

### DISCUSSION THREAD

**Discussion by glecas** [Klocman/Bulk-Crap-Uninstaller#612] [General]
I've contributed: Translation into Swedish.
[BCU Localisation Pack+Sv.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/15366557/BCU.Localisation.Pack%2BSv.zip)

**Comment by Klocman** [comment]
Thank you for the translation! I've created a PR for it in #614. 
If you'd like to test the translation to make sure everything looks good, here's a test build: [net6.0-windows10.0.18362.0.zip](https://github.com/Klocman/Bulk-Crap-Uninstaller/files/15381294/net6.0-windows10.0.18362.0.zip)
I'll wait with merging the translation in the meantime.

**Reply by glecas** [reply]
I've tested it and it looks good 🙂👍
